sections/03_categorize.md

```diff
@@ -28,7 +28,148 @@ we share these data while respecting privacy? I've made a section for each of
+the algorithms could be re-used across tasks. Their aim was to analyze the free
+text portions of pathology reports to identify the primary site and laterality
+of tumors. The only features the authors supplied to the algorithms that they
+evaluated were unigrams and bigrams. These are the counts for single words and
+two-word combinations in a free text document. They subset the full set of words
+and word combinations to the 400 most commonly used ones. The machine learning
+algorithms that they employed (naive bayes, logistic regression, and deep neural
+networks) all performed relatively similarly on the task of identifying the
+primary site. However, when the authors evaluated the more challenging task,
+i.e. evaluating the laterality of each tumor, the deep neural network
+outperformed the other methods. Of particular interest, when the authors first
+trained a neural network to predict primary site and then repurposed those
+features as a component of a secondary neural network trained to predict
+laterality, the performance was higher than a laterality-trained neural network.
+This indicates a potential strength of deep methods. It may be possible to
+repurpose features from task to task, improving overall predictions as the field
+tackles new challenges.
+
+TODO: survival analysis/readmission prediction methods from EHR/EMR style data
+(@sw1 + maybe @traversc). These include:
+* https://github.com/greenelab/deep-review/issues/81
+* https://github.com/greenelab/deep-review/issues/82
+* https://github.com/greenelab/deep-review/issues/152
+* https://github.com/greenelab/deep-review/issues/155
+
+Identifying consistent subgroups of individuals and individual health
+trajectories from clinical tests is also an active area of research. Approaches
+inspired by deep learning have been used for both unsupervised feature
+construction and supervised prediction. In the unsupervised space, early work
+demonstrated that unsupervised feature construction via denoising autoencoder
+neural networks could dramatically reduce the number of labeled examples
+required for subsequent supervised analyses [@doi:10.1101/039800]. A concurrent
+large-scale analysis of an electronic health records system found that a deep
+denoising autoencoder architecture applied to the number and co-occurrence of
+clinical test events, though not the results of those tests, constructed
+features that were more useful for disease prediction than other existing
+feature construction methods [@doi:10.1038/srep26094]. While each of these
+touched on clinical tests, neither considered full text records. Taken together,
+these results support the potential of unsupervised feature construction in this
+domain. However, there are numerous challenges that will need to be overcome
+before we can fully assess the potential of deep learning for this application
+area.
+
+##### Opportunities
+
+However, significant work needs to be done to move these from conceptual
```

Comment:
At this point, bullets are probably good. Still have some more work to do on this but wanted to give @sw1 a chance to integrate his new text.